File: MeTalk/Recorder/Cells/MTHistorySectionHeaderView.m
//
//  MTHistorySectionHeaderView.m
//  MeTalk
//
//  Created by yu chung hyun on 2016. 4. 27..
//  Copyright © 2016년 yu chung hyun. All rights reserved.
//

#import "MTHistorySectionHeaderView.h"
#import "MTMacros.h"

@interface MTHistorySectionHeaderView()
{
    
}
@property (weak, nonatomic) IBOutlet UILabel *lblTitle;

@end

@implementation MTHistorySectionHeaderView

- (void)awakeFromNib
{
    [self.lblTitle setTextColor:ColorFromRGBA(0x00,0x00,0x00,0.5)];
}

- (void)setTitle:(NSString *)str
{
    [self.lblTitle setText:str];
}
@end

File: MeTalk/Recorder/Cells/MTHistorySectionHeaderView.swift
//
//  MTHistorySectionHeaderView.swift
//  MeTalk
//
//  Created by yu chung hyun on 2016. 5. 11..
//  Copyright © 2016년 yu chung hyun. All rights reserved.
//

import UIKit

class MTHistorySectionHeaderView: UITableViewHeaderFooterView {

    @IBOutlet weak var lblTitle: UILabel!

    override func awakeFromNib() {
        self.lblTitle.textColor = UIColor.colorRGB(0x00, 0x00, 0x00,0.5)
    }

    func setTitle(str:String) -> Void {
        self.lblTitle.text = str
    }
    
}
